samples/peripheral/radio_test/README.rst

```diff
@@ -0,0 +1,149 @@
+.. _radio_test:
+
+Radio Test
+##########
+
+The Radio Test sample demonstrates how to configure the radio as a constant RX or TX carrier, a modulated TX carrier, or a RX or TX sweep.
+
+Overview
+********
+
+The tests are controlled with Command Line Interface (CLI) through the serial port.
```

Comment:
What tests? We haven't said anything about tests before. (See comment above)

Can we link to CLI? (Do we have that in NCS?)